src/Components/CreateImageWizard/formComponents/MountPoint.js

```diff
@@ -13,7 +13,7 @@ import path from 'path';
 
 const MountPoint = ({ ...props }) => {
     // check '/' last!
-    const validPrefixes = [ '/home', '/opt', '/srv', '/usr/local', '/var', '/boot', '/' ];
+    const validPrefixes = [ '/home', '/opt', '/srv', '/usr', '/usr/local', '/var', '/app', '/data', '/' ];
```

Comment:
Is /usr/local still a valid prefix since it is a sub directory of a valid prefix? It isn't in the current list:
https://github.com/osbuild/osbuild-composer/blob/main/internal/distro/rhel85/distro.go#L44

Reply:
Any sub directory of a valid prefix should be valid (that was the idea anyway ;) ).

Reply:
So any mounts starting with the prefixes in the `mountpointAllowList`, save for `/`, are still good (see https://github.com/osbuild/osbuild-composer/blob/main/internal/distro/rhel85/distro.go#L418). 

Just to check:
![image](https://user-images.githubusercontent.com/11140201/161248715-38d3380b-19a8-4dfd-98cd-3a669dae67ae.png)

^ this indeed succeeded: 
![image](https://user-images.githubusercontent.com/11140201/161249624-8b1a06ae-450b-4a24-8acf-9353a95dc283.png)